heudiconv/tests/test_regression.py

```diff
@@ -19,16 +19,17 @@
 
 @pytest.mark.parametrize('subject', ['sub-sid000143'])
 @pytest.mark.parametrize('heuristic', ['reproin.py'])
+@pytest.mark.parametrize('anon_cmd', [None, 'anonymize_script.py'])
 @pytest.mark.skipif(not have_datalad, reason="no datalad")
-def test_conversion(tmpdir, subject, heuristic):
+def test_conversion(tmpdir, subject, heuristic, anon_cmd):
     tmpdir.chdir()
     try:
         datadir = fetch_data(tmpdir.strpath, subject)
     except IncompleteResultsError as exc:
         pytest.skip("Failed to fetch test data: %s" % str(exc))
     outdir = tmpdir.mkdir('out').strpath
 
-    args = gen_heudiconv_args(datadir, outdir, subject, heuristic)
+    args = gen_heudiconv_args(datadir, outdir, subject, heuristic, anon_cmd)
```

Comment:
in the light of #509 I now wonder if it all worked out correctly in this PR, since if I get it right - with the anonymization command we should have no longer got file names formatted (below) according to `subject` but rather to anonymized version of it